assets/airports/kjfk.json

```diff
@@ -1,4 +1,5 @@
 {
+    "airac": 1910,
```

Comment:
Line 2: Should be 2001 now, although it'll be 2002 by the time this is merged

Reply:
changed to 2002